db/migrate/20240924090158_add_resubmission_deadline_to_further_informations.rb

```diff
@@ -0,0 +1,5 @@
+class AddResubmissionDeadlineToFurtherInformations < ActiveRecord::Migration[7.2]
+  def change
```

Comment:
is it worth popping that through first seperately, not sure if there are race conditions with pod creations and things

Reply:
We should be ok in that migrations happen before new code is deployed, so so long as we're not making a DB change that makes the DB incompatible with the old code we're safe.